CMakeLists.txt

```diff
@@ -1,104 +1,33 @@
-cmake_minimum_required(VERSION 3.19)
-set(CMAKE_CXX_STANDARD 20)
```

Comment:
Since at the moment there is still a lot of experimentation with which dependencies and features we need i would leave the configuration separate for now and then unify them once things get more stable.